src/pages/test-results-search/components/search-result/search-result.scss

```diff
@@ -3,12 +3,7 @@ search-result {
     margin-left: 0;
     margin-right: 0;
     width: 100%;
-  }
-  .indicator {
-    background: rgba(155, 155, 155, 0.5);
-    position: absolute;
-    height: 120px;
-    width: 8px;
+    border-left: 8px solid rgba(87, 73, 73, 0.5);
```

Comment:
Should this be a standard colour or declared in the variables mixin?

Reply:
This is `gds-black`, so could be switched to that instead.

Reply:
Have updated this to the hex version , which isn't one of our standard colours will check with Nad when he's back in and see if he's happy for us to use one of them